### ISSUE TITLE
Can I replace OpenAI with alternatives?

### ISSUE THREAD

**Issue by 258Paco** [microsoft/lida#134]
Hi, I am working on this project to implement LLMs to my company's control tower. I found this to be a very suitable model for implementation. However, I can not access open ai in my country, and I am planning to use alternatives for the implementation, is it possible to do so? If so, any model recommendations that I can use?

**Comment by sha-linimoorthy** [comment]
Yes you can use models from providers like HuggingFace, palm, cohere.
